spec/integrations/instrumentation/datadog_metrics_flow_spec.rb

```diff
@@ -3,7 +3,7 @@
 # Here we subscribe to our listener and make sure nothing breaks during the notifications
 # We use a dummy client that will intercept calls that should go to DataDog and check basic
 # metrics presence
-require 'karafka/instrumentation/vendors/datadog/listener'
+require 'karafka/instrumentation/vendors/datadog/metrics_listener'
```

Comment:
Would be good to also add spec to the legacy one "just in case" as long as we have it.

Reply:
Makes sense, what do you think of https://github.com/karafka/karafka/pull/1341/commits/4965722f259a910983140ed6778a83b65c3ee804?w=1?

I didn't want to duplicate all the code in a different spec file since we are planning to deprecate the class soon.

Reply:
Its ok :)